samples/bot-formatting-cards/nodejs/Readme.md

```diff
@@ -120,7 +120,7 @@ The simplest way to run this sample in Teams is to use Teams Toolkit for Visual
         ```
 
   **Note: In adaptive card, what we are defining (User details) should be exist in the same tenant where you are testing the app (teams' login) etc...**
-  - Update the user AAD object ID in your adaptive card JSON from your tenant's AAD users available in the Azure portal.
+  - Update the user Microsoft Entra object ID in your adaptive card JSON from your tenant's Microsoft Entra users available in the Azure portal.
```

Comment:
Microsoft Entra users >> Microsoft Entra ID users

Reply:
fixed In First PR